source/Extensibility.Authentication/Web/EncodedQueryStringParser.cs

```diff
@@ -31,8 +31,8 @@ string DecodeValue(string encodedValue)
 
         public class QueryStringParameter
         {
-            public string Name { get; set; }
-            public string Value { get; set; }
+            public string Name { get; set; } = string.Empty;
```

Comment:
Would it be better to initialize these via the `ctor`?

Reply:
Yes, changed 👍